### ISSUE TITLE
Loss does not converge

### ISSUE THREAD

**Issue by liliangqi** [BinWang-shu/yolo_v3_pytorch#1]
Hi, I tried to train the model with your code but when testing, the output looks like http://oxg3si0yz.bkt.clouddn.com/wrong_det1.jpg. I think there are some problems during training. Did you get good result by this model?

**Comment by BinWang-shu** [comment]
Yes, there are some problems. I'm also looking for the where are wrong. Thanks for attempting my code and if you can find the problems, please tell me. Thank you!!

**Comment by BobLiu20** [comment]
any updates? thanks.

**Comment by BinWang-shu** [comment]
sorry, I'm busy with a project recently. I will try my best to update it.

**Comment by BinWang-shu** [comment]
I have updated the code and I have successfully got the results.

**Comment by themathgeek13** [comment]
Hi @BinWang-shu, how many epochs did it require to give reasonable object detection and classification results? I have run with a batch size of 1 for 30 epochs but it doesn't give sensible results yet.

**Comment by BinWang-shu** [comment]
I got the sensible results when 100 epochs with a batch size of 1 and you need ensure the balance between the samples to get the reasonable classification.

**Comment by BinWang-shu** [comment]
I have train it on VOC2007 dataset. But I do not balance the samples so the results of classification are not good.

**Comment by 1239238241** [comment]
Hi, I am not sure what do you mean by 'balance the samples'
